test-resources/java-default-project/.bazelbsp/bazelbsp.trace.json

```diff
@@ -0,0 +1,327 @@
+[Trace - 06:46:09 PM] Received request 'build/initialize - (1)'
```

Comment:
prob this file is unnecessary, as well as other logs

Reply:
Thanks! Didn't notice they were there, will make sure the gitignores are updated as well